Data/Uniques/belt.lua

```diff
@@ -224,6 +224,35 @@ Requires Level 40
 {variant:2}(20-30)% increased Damage with Hits against Rare monsters
 When you Kill a Rare monster, you gain its mods for 20 seconds
 ]],[[
+Hyperboreus
+Leather Belt
+Variant: Life Regen
+Variant: Cold and Chaos Resistances
+Variant: Lightning and Chaos Resistances
+Variant: Fire and Chaos Resistances
+Variant: Strength and Intelligence
+Variant: Dexterity and Intelligence
+Variant: Strength and Dexterity
+Variant: Trap Throwing Speed
+Variant: Energy Shield Regen
+Variant: Lucky Crit Chance while Focussed
+Requires Level 60
++(25–40) to maximum Life
++(30–40)% to Cold Resistance
+Chill nearby Enemies when you Focus, causing 30% reduced Action Speed
+Focus has (15–25)% increased Cooldown Recovery Speed
+(50–70)% increased Damage with Hits and Ailments against Chilled Enemies
+{variant:1}{crafted}(1-2)% of Life Regenerated per second during any Flask Effect
```

Comment:
```suggestion
{variant:1}{crafted}(1.0-2.0)% of Life Regenerated per second during any Flask Effect
```
Allows slider to work correctly for fractions. This mod can have the value of 1.5